### PR TITLE
[15.0][IMP] stock_inventory: do not delete inv adjustment groups if in process or done

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
I think it is weird you can delete inventory adjustments groups that are in progress or completed, I think they should be kept for traceability to what was done.
@ForgeFlow 
@DavidJForgeFlow

### CODE REVIEW

File: stock_inventory/models/stock_inventory.py [hunk 1]
```diff
@@ -214,3 +214,14 @@ def _check_one_product_in_product_selection(self):
                             " you are only able to add one product."
                         )
                     )
+
+    def unlink(self):
+        for adjustment in self:
+            if adjustment.state != "draft":
+                raise UserError(
+                    _(
+                        "You can only delete inventory adjustmnets groups in"
```
Comment (rousseldenis) [inline stock_inventory/models/stock_inventory.py]:
```suggestion
                        "You can only delete inventory adjustments groups in"
```

**Review by rousseldenis — APPROVED** [review]
Makes sense as it was the case in v<=14

https://github.com/odoo/odoo/blob/14.0/addons/stock/models/stock_inventory.py#L83

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM!

Maybe you could also add some ondelete="retrict" for stock.move.lines.inventory_id?

**Comment by AaronHForgeFlow** [comment]
@LoisRForgeFlow  Nice suggestion, thanks

**Pull request merged**